mod/developers/views/default/developers/ajax_demo.js

```diff
@@ -0,0 +1,70 @@
+define(function(require) {
+	var Ajax = require('elgg/Ajax');
```

Comment:
Is there a reason `Ajax` starts with capital letter? AFAIK all other module names have only lowercase letters.